src/core/utils/useragent.js

```diff
@@ -13,3 +13,38 @@ export function isMicrosoftEdge () {
     return navigator.userAgent.includes('Edg');
   }
 }
+
+/**
+ * Returns whether the current browser is Safari.
+ * Currently, Safari does not support User-Agent clients hints (userAgentData).
+ * Chrome likes to pretend that it's Safari.
+ *
+ * @returns {boolean}
+ */
+export function isSafari () {
+  if (isMicrosoftEdge()) {
```

Comment:
How come we need to check for microsoft edge here?

Reply:
Edge also likes to pretend it's safari

Reply:
Adding to the jsdoc!